backend/routes/loginRoute.js

```diff
@@ -0,0 +1,64 @@
+const asyncHandler = require("express-async-handler");
```

Comment:
The use for `await` on bcrypt shouldn't be required unless bcrypt returns us a promise

Reply:
The file loginRoute.js is not needed and should be removed

Reply:
Fixed: Changes are reflected in recent commits (File removed)